x/gov/test_common.go

```diff
@@ -146,3 +146,25 @@ func SortByteArrays(src [][]byte) [][]byte {
 	sort.Sort(sorted)
 	return sorted
 }
+
+func testProposal() TextProposal {
+	return NewTextProposal("Test", "description")
+}
+
+// checks if two proposals are equal
+func ProposalEqual(proposalA Proposal, proposalB Proposal) bool {
+	if proposalA.ProposalID == proposalB.ProposalID &&
```

Comment:
instead of testing all the fields, we should simply marshal both proposals and test if the bytes are equal, simplified the code, makes it redundant against field additions. (also we're not so concerned about efficiency as this is used for testing)